File: app/src/main/java/com/pondipb/pondibrian/ukulimaiq/MenuSelection.java
package com.pondipb.pondibrian.ukulimaiq;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class   MenuSelection extends AppCompatActivity  implements View.OnClickListener {
    //Buttons Added
    Button btnAgronomy;
    Button btnClimate;
    Button btnMarketPrice;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_menu_selection);

        btnAgronomy = findViewById(R.id.btnAgronomy);
        btnAgronomy.setOnClickListener(this);// calling onClick() method
        btnClimate = findViewById(R.id.btnClimate);
        btnClimate.setOnClickListener(this);
        btnMarketPrice = findViewById(R.id.btnMarketPrice);
        btnMarketPrice.setOnClickListener(this);


    }

    @Override
    public void onClick(View view) {


        switch (view.getId()) {

            case R.id.btnAgronomy:
                // intent method call.
                openAgronomyActivity();
                break;

            case R.id.btnClimate:
                // intent method call
                openClimateActivity();
                break;

            case R.id.btnMarketPrice:
                // intent Method call
                openPriceActivity();

                break;


            default:
                break;

        }

    }

    private void openClimateActivity() {
        Intent intent = new Intent(this, ClimateActivity.class);
        startActivity(intent);
    }

    private void openAgronomyActivity() {
        Intent intent = new Intent(this, AgronomyActivity.class);
        startActivity(intent);
    }

    private void openPriceActivity() {
        Intent intent = new Intent(this, PriceActivity.class);
        startActivity(intent);
    }
    //Method to for Community Map button

}

File: app/src/main/kotlin/com/pondipb/pondibrian/ukulimaiq/MenuSelection.kt
package com.pondipb.pondibrian.ukulimaiq

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MenuSelection : AppCompatActivity(), View.OnClickListener {
    private lateinit var btnAgronomy: Button
    private lateinit var btnClimate: Button
    private lateinit var btnMarketPrice: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu_selection)

        btnAgronomy = findViewById(R.id.btnAgronomy)
        btnAgronomy.setOnClickListener(this)
        
        btnClimate = findViewById(R.id.btnClimate)
        btnClimate.setOnClickListener(this)
        
        btnMarketPrice = findViewById(R.id.btnMarketPrice)
        btnMarketPrice.setOnClickListener(this)
    }

    override fun onClick(view: View) {
        when (view.id) {
            R.id.btnAgronomy -> openAgronomyActivity()
            R.id.btnClimate -> openClimateActivity()
            R.id.btnMarketPrice -> openPriceActivity()
        }
    }

    private fun openClimateActivity() {
        val intent = Intent(this, ClimateActivity::class.java)
        startActivity(intent)
    }

    private fun openAgronomyActivity() {
        val intent = Intent(this, AgronomyActivity::class.java)
        startActivity(intent)
    }

    private fun openPriceActivity() {
        val intent = Intent(this, PriceActivity::class.java)
        startActivity(intent)
    }
}
